backend/tests/unit/agents/test_base_agent.py

```
@@ -1292,3 +1292,192 @@ async def test_execute_mcp_tools_backward_compatibility_no_conversation(self, mo
+        self, mock_llm_manager, mock_mcp_client, mock_mcp_registry
+    ):
+        """Test that process_alert returns FAILED result when execution exceeds timeout."""
+        import asyncio
+        from tarsy.models.constants import StageStatus
+        from tarsy.config.settings import get_settings
+        
+        # Create agent
+        agent = TestConcreteAgent(mock_llm_manager, mock_mcp_client, mock_mcp_registry)
+        
+        # Mock _process_alert_impl to take longer than timeout
+        async def slow_implementation(*args, **kwargs):
```

Ruff (0.14.10)

Unused function argument: `args`

(ARG001)